CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). (Patch version X.Y.0 is implied if not specified.)
 
 ## [Unreleased](https://github.com/usgs/aqts-capture-discrete-loader/compare/master..aqts-capture-discrete-loader-0.0.1)
```

Comment:
The discrete loader is on a later version than this, I think you want 0.1.12 (most recent tagged release)

Reply:
fixed